### PR TITLE
#159053963 Feature create comments 

### PR DESCRIPTION
**Pull request by migot01**
#### What does this PR do?
Setup create comments feature

#### Description of tasks to be completed
Ensure that users can create comments and threaded comments

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans.git
```
```sh
$ git checkout -b ft-create-comments-159053963
```
```sh
$ cd ah-titans/authors
```

> Add **.env** file and copy **example.env** content.
> Add your local database configuration settings to your **.env** file
```sh
$ python3.6 -m venv venv
```
```sh
$ source venv/bin/activate
```
```sh
$ source .env
```
```sh
$ pip install -r requirements.txt
```
```sh
$ python manage.py makemigrations
```
```sh
$ python manage.py runserver
```
```sh
Using postman, send a post request to the URL http://127.0.0.1:8000/api/articles/comments/
to create a comment on an article and http://127.0.0.1:8000/api/articles/comments/<comment_pk>/
to comment on a comment
Refer to the screenshots below.
```

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#159053963](https://www.pivotaltracker.com/story/show/159053963)

####  comment on an article
image

#### Create threaded comment
image
#### Get Comments
image

### CODE REVIEW

**Comment by ridgekimani** [comment]
Please fix the merge conflicts @migot01

**Review by janetnim — COMMENTED** [review]
Order of imports in models.py
```
from authors.apps.core.models import TimestampModel
```

File: authors/tests/articles/test_comments.py [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    def test_cannot_delete_inexisting_comment(self):
+        """Test user cannot  delete inexisting comment """
+        token = self.login_verified_user(self.test_user)
+        self.create_article(token, self.article)
+        self.create_comment(token, 'tests', self.comment)
+        response = self.delete_comment(token, 'test',self.comment,'2')
+        self.assertIn("A comment with this ID does not exist.",
+                      response.content.decode())
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
```
Comment (petermwash) [inline authors/tests/articles/test_comments.py:253]: Add an end of line.

File: authors/apps/articles/renderers.py [hunk 2]
```diff
@@ -33,3 +33,23 @@ def render(self, data, media_type=None, renderer_context=None):
...
+                return json.dumps({
+                    'comment': data
+                })
+            return json.dumps({
+                'comments': data
+            })
+        return json.dumps({
+                'comment': 'No article found.'
+            })
+
```
Comment (ronaldndirangu) [inline authors/apps/articles/renderers.py]: Please add an end of line to this.

**Review by ronaldndirangu — APPROVED** [review]
Good work. I think it is ready to merge.

**Pull request merged**